nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Utilities/MSBuildHelperTests.cs

```diff
@@ -1669,6 +1666,59 @@ await UpdateWorkerTestBase.MockNuGetPackagesInDirectory([
         };
         AssertEx.Equal(expectedResolvedDependencies, actualResolvedDependencies);
     }
+
+    [Fact(Timeout = 120_000)] // 2m
```

Comment:
Explicitly added a timeout because without the fix, this test will hang.  Normal runtime is <3s.